bin/run-docker

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Is this actually a build file for use *inside* docker?  If so maybe it should be named differently, like with a leading underscore (what I've used for some scripts only intended to be run by make) or just put elsewhere (I'm not particularly happy with the leading underscore approach)